includes/class-wcpdf-admin.php

```diff
@@ -675,9 +675,13 @@ public function save_invoice_number_date($post_id, $post) {
 
 			$order = wc_get_order( $post_id );
 			if ( $invoice = wcpdf_get_invoice( $order ) ) {
-				$is_new = false === $invoice->exists();
-				$_POST = stripslashes_deep( $_POST );
-				$document_data = $this->process_order_document_form_data( $_POST, $invoice->slug );
+				$is_new        = false === $invoice->exists();
+				$request_data  = stripslashes_deep( $_POST );
```

Comment:
Perhaps rename this to `$form_data` as this is the parameter name we use in the `process_order_document_form_data` function.
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/5048ca51bcfed8db1eba4e0f5ae72ecdbf01bfa6/includes/class-wcpdf-admin.php#L952